miso-web/src/main/webapp/pages/adminsub.jsp

```diff
@@ -29,8 +29,6 @@
 	<div class="menu">
 		<a href="<c:url value="/miso/sample/receipt"/>">Receive
```

Comment:
Please also remove this line, as the sampleReceipt JSP is deleted :)

Reply:
Weird - I thought I had. It's gone now